pkg/seeds/seeds.go

```diff
@@ -75,14 +75,39 @@ func SeedRepositoryConfigurations(db *gorm.DB, size int, options SeedOptions) er
 	return nil
 }
 
-func randomLastRead() (lastReadTime *time.Time, lastReadError *string) {
-	if rand.Int()%2 == 0 {
-		return nil, nil
+func randomIntrospectionStatusMetadata() (
+	lastIntrospectionTime,
+	lastIntrospectionSuccessTime,
+	lastIntrospectionUpdateTime *time.Time,
+	status string,
+	lastIntrospectionError *string) {
+	var (
+		state     int
+		timestamp time.Time
+	)
+	state = rand.Intn(4)
+	timestamp = time.Now()
+
+	switch state {
+	case 0:
+		status = "Pending"
```

Comment:
I couldn't use the constants defined in the dao package here, because then there'd be a circular dependency. I'm not sure where else I could put the constants. I'm open to suggestions :)

Reply:
config/ maybe?  I know its not great to put it there conceptually, but config can basically be used by everything.

Thats where i put the constants for distribution versions and arch